src/components/Views/ObservationEditor/ObservationEditor.js

```diff
@@ -379,6 +379,10 @@ class ObservationEditor extends React.Component<
         navigation.navigate({
```

Comment:
instead of all this code, since it seems like it's doing the same thing, you could probably just call `this.handleSave`

Reply:
oooh smaht!